plugins/osdn/ovs/bin/openshift-sdn-ovs

```diff
@@ -76,6 +78,15 @@ add_ovs_flows() {
 	ovs-ofctl -O OpenFlow13 add-flow br0 "table=7, priority=100, reg0=0, ip, nw_dst=${ipaddr}, actions=output:${ovs_port}"
 	ovs-ofctl -O OpenFlow13 add-flow br0 "table=7, priority=100, reg0=${tenant_id}, ip, nw_dst=${ipaddr}, actions=output:${ovs_port}"
     fi
+
+    if [ -n "${ingress_bw}" ]; then
+        ovs-vsctl set port ${veth_host} qos=@newqos -- --id=@newqos create qos type=linux-htb other-config:max-rate=${ingress_bw}
```

Comment:
The new syntax we're not using anywhere else yet is made even more confusing by the fact that `@newqos` gets used before it's set (which is apparently allowed, but... weird). Could you do the `create` first and then the `set`?

Also, something somewhere (commit message?) should document why linux-htb rather than any of the others